8 patch review

erinmd opened the pull request.

HannahBurks reviewed (changes requested):
Hey Erin, this is great, I would however look to add a path that generically handles 404 on app.js - have a look on how you can do this :) Once added send another PR

erinmd commented:
I've added a generic 404 path, and refactored the other 404 errors.
This was from a previous one, but not sure if lines 193 and 202 in the post request tests should be 400/404 errors. I had them as 400 before, but should it be 404 as they don't exist instead?
Thanks for the feedback!

Geo-Beard reviewed (commented):
Looks good overall. Feel free to merge and move on!
